plugins/scanners/scanoss/src/test/kotlin/ScanOssScannerDirectoryTest.kt

```diff
@@ -110,4 +118,62 @@ class ScanOssScannerDirectoryTest : StringSpec({
             )
         }
     }
+
+    "Scanner should exclude only files matching the specified path pattern (**/*.kt)" {
+        val pathExcludes = listOf(
+            PathExclude(
+                pattern = "**/*.kt", // Glob pattern to match all .kt files in any directory
+                reason = PathExcludeReason.BUILD_TOOL_OF,
+                comment = "Excluding .kt source files from scanning"
+            )
+        )
+
+        // Verify our test file exists - this file should be included in the scan
+        // since it doesn't match the exclusion pattern (it's a .go file, not .kt)
+        val includedFile = File(EXCLUSION_TEST_DIRECTORY, "server.go")
+        if (!includedFile.exists()) {
+            fail("file ${includedFile.absolutePath} does not exist")
+        }
```

Comment:
Is the `includedFile` variable and this whole check really required? Can we just rely on the

    includedFiles.any { it.contains("server.go") } shouldBe true

assertion in line 177 to fail?

Reply:
The check is verifying that the test file `server.go` exists in the test directory before we run the scanner. 

This is intentional because if the file doesn't exist, the later test (which verifies that the scanner includes this file in its scan) would fail for the wrong reason - not because the scanner logic is broken, but because the test file is missing.

This check acts as a pre-condition for the test rather than testing the scanner itself. Later in the test (line 177), we verify that the scanner correctly processes this file using the `includedFiles.any { it.contains("server.go") }` assertion.

Would you prefer a different approach to handle this pre-condition check?

Reply:
> Would you prefer a different approach to handle this pre-condition check?

No, it's fine.